exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_exporter_options.h

```diff
@@ -57,6 +56,30 @@ struct OtlpHttpExporterOptions
   // Requests per connections
   std::size_t max_requests_per_connection = 8;
 #endif
+
+#ifdef ENABLE_OTLP_HTTP_SSL
+  bool ssl_insecure_skip_verify{false};
+
+  std::string ssl_ca_cert_path   = GetOtlpDefaultTracesSslCertificatePath();
+  std::string ssl_ca_cert_string = GetOtlpDefaultTracesSslCertificateString();
+
+  std::string ssl_client_key_path   = GetOtlpDefaultTracesSslClientKeyPath();
+  std::string ssl_client_key_string = GetOtlpDefaultTracesSslClientKeyString();
```

Comment:
Instead of having both CA and client certificate, can we just have client certificate file/path. The user can decide whether to send single client certificate, or send the client certificate chain (if there are intermediate certificates) in the single file/path.

Reply:
Ok, just to be clear, we are using - 

 OTEL_EXPORTER_OTLP_CERTIFICATE -> To set the trusted CA certificate(s) which would be used to authenticate the collector during the SSL handshake.
OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE - To set the client certificate(or certificate chain) which would be used to authenticate otlp-http client during the SSL handshake.

It looks good if this is the case. Sorry for the confusion, will go through the full PR :)

Reply:
@lalitb Correct - auth is done both ways (mTLS)